### PR TITLE
C 23

### LINKED ISSUE #23: Create new branch to show faculty courses
We will need to create a new branch (protected) that is based on the `C-22` branch, however, we need to strip it back to just include the course information that an instructor is teaching for each semester.

### PR DESCRIPTION
**Pull request by nickwoodle**
So for this PR, I have created a new protected branch called faculty (which is based on master). The branch I am merging is based on `C-22`, so this has all of Sam's work include, minus anything that needed to be removed for the faculty version of this.

Closes #23

### CODE REVIEW

**Comment by nickwoodle** [comment]
Please review #25 before this one please :grin:

**Review by amgoodfellow — APPROVED** [review]
These changes LGTM

**Comment by amgoodfellow** [comment]
Sorry about that - I requested re-review for myself. This was not the PR we were talking about in the meet lol

File: package.json [hunk 1]
```diff
@@ -2,21 +2,20 @@
...
   "version": "1.0.0",
   "private": true,
+  "proxy": "http://localhost:8090",
   "dependencies": {
-    "@material-ui/core": "~3.6.2",
-    "@material-ui/icons": "^3.0.2",
+    "@fullcalendar/daygrid": "^5.1.0",
+    "@fullcalendar/list": "^5.1.0",
+    "@fullcalendar/react": "^5.1.0",
+    "@fullcalendar/timegrid": "^5.1.0",
```
Comment (joshwargo-ou) [inline package.json]: Are you using the calendar stuff?  (If not, this can be ripped out.)

Comment (nickwoodle) [reply package.json]: This has been removed.

File: src/Calendar.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
+@media only screen and (max-width: 600px) {
```
Comment (joshwargo-ou) [inline src/Calendar.css]: Based on the name of this CSS file, I would assume we don't need it (if it indeed was for the Calendar component).

Comment (nickwoodle) [reply src/Calendar.css]: This has been removed

**Comment by joshwargo-ou** [comment]
This LGTM

**Review by joshwargo-ou — APPROVED** [review]

**Comment by sbracellari** [comment]
This looks good to me, the only thing I noticed was some console warnings:

image

Not a showstopper tho

**Review by sbracellari — APPROVED** [review]
LGTM

**Review by amgoodfellow — APPROVED** [review]
This LGTM :100: 

Nice job converting this over :smiley:

**Pull request merged**